packages/dx-react-scheduler/docs/reference/current-time-indicator.md

```diff
@@ -0,0 +1,56 @@
+# CurrentTimeIndicator Plugin Reference
+
+A plugin that renders the Scheduler's current time indicator
+
+## Import
+
+Use the following statement to import a plugin with embedded theme components:
+
+```js
+import { CurrentTimeIndicator } from '@devexpress/dx-react-scheduler-material-ui';
+```
+
+You can import the themeless plugin if you want to use custom components:
+
+```js
+import { CurrentTimeIndicator } from '@devexpress/dx-react-scheduler';
+```
+
+## User Reference
+
+### Dependencies
+
+- [DayView](day-view.md) [Optional]
+- [WeekView](week-view.md) [Optional]
+- [MonthView](month-view.md) [Optional]
+- [Appointments](appointments.md) [Optional]
+- [DragDropProvider](drag-drop-provider.md) [Optional]
+
+### Properties
+
+Name | Type | Default | Description
+-----|------|---------|------------
+updateInterval | number | 60000 | The time interval in milliseconds between Indicator's position updates.
+reduceBrightnessOfPastAppointments | boolean | `false` | Specifies whether the brightness of past appointments should be reduced.
+shadePastCells | boolean | `false` | Specifies whether past cells should be shaded.
```

Comment:
I think we should make these properties names similar. `shadePastCelss`, `shadePastAppointments` or something like that